react/react1/week2/to-do-list/src/Components/Timer.js

```diff
@@ -0,0 +1,13 @@
+import { useState, useEffect } from "react";
+
+export const Timer = () => {
+  const [count, setCount] = useState(0);
+
+  useEffect(() => {
+    setTimeout(() => {
```

Comment:
Does this work with `setTimeout` or should you use `setInterval`? The former will only execute once, after 1 second, while the latter will run periodically, continuously updating the `count`